wasm_samples/multisig_accounts/build.rs

```diff
@@ -0,0 +1,27 @@
+//! Compile binary containing common logic to each multisig account for handling multisig transactions
+
+use std::{io::Write, path::Path};
+
+const TRIGGER_DIR: &str = "../multisig_transactions";
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    for path in [
+        TRIGGER_DIR,
+        "../../crates/iroha_data_model/src",
+        "../../crates/iroha_core/src/smartcontracts",
+        "../../crates/iroha_smart_contract/src",
```

Comment:
I don't think this is needed. We could put this into all smart contracts then? @0x009922

Reply:
Yeah, this list of dependencies is not strictly exhaustive.

IMO it's better to run these build scripts unconditionally - `iroha_wasm_builder` relies on `cargo`, thus rebuilds only when necessary.

_However_, it's possible to extend the wasm builder to read [Cargo messages](https://docs.rs/cargo_metadata/0.18.1/cargo_metadata/enum.Message.html), track all involved dependencies, and print them with `cargo::rerun-if-changed={path}`. Sounds like an overkill to me though.

Reply:
Deleted the entire `multisig_*/build.rs`. Instead, made the script build them in sequence